ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/MLAG_IPV6.yml

```diff
@@ -18,6 +18,12 @@ l3leaf:
         - name: MLAG_IPV6_L3LEAF1A
           id: 3
           mgmt_ip: 192.168.201.116/24
+          underlay_multicast:
+            pim_sm:
+              mlag: false
```

Comment:
I am not seeing any config changes from this. Why do we need this input? Did we change a default somehow?